measurement/validation.py

```diff
@@ -133,45 +109,19 @@ def flag_suspicious_data(
         data: The dataframe with the data to be evaluated.
         maximum: The maximum allowed value.
         minimum: The minimum allowed value.
-        period: The expected period for the measurements, in minutes.
         allowed_difference: The allowed difference between the measurements.
 
     Returns:
         A dataframe with the suspicious data.
     """
-    time_lapse = flag_time_lapse_status(data, period)
     value_difference = flag_value_difference(data, allowed_difference)
     value_limits = flag_value_limits(data, maximum, minimum)
-    return pd.concat([time_lapse, value_difference, value_limits], axis=1)
-
-
-def flag_suspicious_daily_count(
-    data: pd.Series, period: Decimal, null_limit: Decimal
-) -> pd.DataFrame:
-    """Finds suspicious records count for daily data.
-
-    Args:
-        data: The count of records per day.
-        period: The expected period for the measurements, in minutes.
-        null_limit: The percentage of null data allowed.
-
-    Returns:
-        A dataframe with the suspicious data.
-    """
-    expected_data_count = 24 * 60 / float(period)
```

Comment:
The long and nicely crafter answer I had prepared boils down to this:

We cannot calculate the expected data count anymore because we don't have the `period` but we can have an estimate of what the expected count would be by using the `mode` of the daily count (i.e. the most common value for the number of counts). It would be expected that most days would have the right number of entries and only some would deviate, so we could do:

```
expected_data_count = data.mode().iloc[0]
```

assuming that `data` is a series. If it where a dataframe, it would be `iloc[0, 0]`

So, recover this function and use the above as the `expected_data_count`, please.